https://github.com/pocl/pocl/commit/a13cb332d6678d4556d7319b284b77c371c4b91e

commit a13cb332d6678d4556d7319b284b77c371c4b91e
Author: Martin Kletzander <nert.pinx@gmail.com>
Date:   Tue Jan 17 09:04:41 2023 +0100

    Add stub macro POCL_MSG_PRINT_ALMAIF_MMAP without POCL_DEBUG_MESSAGES
    
    Without this the build fails with the following error when built without
    POCL_DEBUG_MESSAGES:
    
    ../lib/CL/devices/almaif/MMAPRegion.cc: In constructor ‘MMAPRegion::MMAPRegion(size_t, size_t, int)’:
    ../lib/CL/devices/almaif/MMAPRegion.cc:43:3: error: ‘POCL_MSG_PRINT_ALMAIF_MMAP’ was not declared in this scope; did you mean ‘POCL_MSG_PRINT_ALMAIF2’?
       43 |   POCL_MSG_PRINT_ALMAIF_MMAP(
          |   ^~~~~~~~~~~~~~~~~~~~~~~~~~
          |   POCL_MSG_PRINT_ALMAIF2
    
    and about 10 more.

--- a/lib/CL/pocl_debug.h
+++ b/lib/CL/pocl_debug.h
@@ -284,6 +284,7 @@ POCL_EXPORT
 
     #define POCL_MSG_PRINT_ALMAIF2(...)  do {} while (0)
     #define POCL_MSG_PRINT_ALMAIF(...)  do {} while (0)
+    #define POCL_MSG_PRINT_ALMAIF_MMAP(...)  do {} while (0)
     #define POCL_MSG_PRINT_PROXY2(...)  do {} while (0)
     #define POCL_MSG_PRINT_PROXY(...)  do {} while (0)
     #define POCL_MSG_PRINT_VULKAN2(...)  do {} while (0)
